cmd/run

```diff
@@ -16,7 +16,7 @@ sudo rm -rf inst/reports inst/run-port-* $cmdrun_log
 sudo chown -f $USER -R inst || true
 
 if [ -n "$build_tests" ]; then
-    cmd/build
+    cmd/build check
```

Comment:
actually, now that I look at it, this logic is wrong.  By definition, -b should run cmd/build -- it should actually build the tests.   The pulling images from docker behavior is broken right now, so can't be enabled by default until it's properly planned out with a complete build flow...